server/command/info.go

```diff
@@ -13,12 +13,15 @@ import (
 func (s *service) executeInfo(params *params) (*model.CommandResponse, error) {
 	conf := s.conf.GetConfig()
 	resp := md.Markdownf("Mattermost Apps plugin version: %s, "+
-		"[%s](https://github.com/mattermost/%s/commit/%s), built %s\n",
+		"[%s](https://github.com/mattermost/%s/commit/%s), built %s, Cloud Mode: %t, Developer Mode: %t\n",
```

Comment:
Somewhere the mode string is generated (something like "Cloud Developer" or something like that), I think for some log in the initialization. Would that be more user friendly? Or do we prefer the true/false statements?

Reply:
I choose the boolean because it given you the full list of possible permutations. That help with debugging as you don't have to thing about which other options are possible.